### ISSUE TITLE
check the swish activation

### ISSUE THREAD

**Issue by tobigithub** [josharnoldjosh/massspectra#24]
Searching for Activation Functions
https://arxiv.org/abs/1710.05941

https://github.com/ChingChuan-Chen/keras_swish_beta and 
https://github.com/dataplayer12/swish-activation

````
from keras import backend as K
from keras.utils.generic_utils import get_custom_objects

class Swish(Activation):
    
    def __init__(self, activation, **kwargs):
        super(Swish, self).__init__(activation, **kwargs)
        self.__name__ = 'swish'

def swish(x):
    return (K.sigmoid(x) * x)

get_custom_objects().update({'swish': Swish(swish)})
````

**Comment by tobigithub** [comment]
Swish: http://www.wolframalpha.com/input/?i=plot+(sigmoid+x)+*+x

![image](https://user-images.githubusercontent.com/2569565/44898008-81677f80-acb2-11e8-8757-1ba3523b8b1f.png)
